### PR TITLE
Test case - RBD thick provisioned PVC

### PR DESCRIPTION
**Pull request by jilju**
Test case to verify PVC creation and consumption using RBD thick provisioning enabled storage class.
OCS-2500 - FT - PVC creation using RBD thick provisioning enabled storage class
Signed-off-by: Jilju Joy <jijoy@redhat.com>

### CODE REVIEW

**Review by wusui — APPROVED** [review]
Looks good.  I would like to know what the pvc_sizes will be once the other bug is fixed.

**Comment by jilju** [comment]
> Looks good. I would like to know what the pvc_sizes will be once the other bug is fixed.
Added PVC size 5Gi

File: tests/manage/pv_services/test_rbd_thick_provisioning.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                size=pvc_size,
+                access_modes=access_modes_rbd,
+                status=constants.STATUS_BOUND,
+                num_of_pvc=3,
+                timeout=120,
+            )
+            for pvc_obj in pvc_objs:
+                pvc_obj.io_file_size = pvc_and_file_sizes[pvc_size]
+                pvc_obj.storage_type = (
+                    "block" if pvc_obj.get()["spec"]["volumeMode"] == "Block" else "fs"
```
Comment (yitzhak12) [inline tests/manage/pv_services/test_rbd_thick_provisioning.py:72]: I see the words "block" and "fs" lots of time in the pv tests. I think we can replace "block" and "fs" with the constants `STORAGE_TYPE_BLOCK`, and `STORAGE_TYPE_FS` or similar names.

**Review by yitzhak12 — COMMENTED** [review]
I am not 100% familiar with the pv tests. But from what I see except my comment, the PR looks great.

File: tests/manage/pv_services/test_rbd_thick_provisioning.py (hunk 1 above)
Comment (jilju) [reply tests/manage/pv_services/test_rbd_thick_provisioning.py]: Done

**Review by yitzhak12 — APPROVED** [review]

**Review by Avilir — APPROVED** [review]
LGTM

**Review by ebenahar — APPROVED** [review]

**Pull request merged**